site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2782,6 +2783,36 @@ public function removeFromSeekingTeam($g_id, $user_id) {
         $this->course_db->query("DELETE FROM seeking_team WHERE g_id=? AND user_id=?", [$g_id, $user_id]);
     }
 
+    /**
+     * Edit the user's message from table seeking_team
+     *
+     * @param string $g_id
+     * @param string $user_id
+     * @param string $message
+     */
+    public function updateSeekingTeamMessageById($g_id, $user_id, $message) {
+        $this->course_db->query("UPDATE seeking_team SET message=? WHERE g_id=? AND user_id=?", [$message, $g_id, $user_id]);
+    }
+
+    /**
+     * Edit the user's message from table seeking_team
+     *
+     * @param string $g_id
+     * @param string $user_id
+     * @return string $user_message
+     */
+    public function getSeekMessageByUserId($g_id, $user_id) {
+        $this->course_db->query(
```

Comment:
Update the php-docs.

Reply:
I really liked the suggestions, I think I have them implemented right. The user can now set a message. If one is set then you can edit the message in the text area or remove it entirely.